README.md

```diff
@@ -3,3 +3,28 @@
 [![GoDoc](https://pkg.go.dev/badge/github.com/owasp-amass/open-asset-model/?utm_source=godoc)](https://pkg.go.dev/github.com/owasp-amass/open-asset-model)
 
 Asset definitions for an organization's external attack surface
+
+## Assets
+
+Supported asset types:
+
+| Asset | Type definition |
+|:------|:-----------|
+| Fully Qualified Domain Name | `FQDN` |
+| Autonomous System | `AutonomousSystem` |
+| Regional Internet Registry Organization | `RIROrganization` |
+| IP Address | `IPAddress` |
+| Netblock | `Netblock` |
+
+## Documentation
+
+The documentation can be found in the Go packages repository:[open-asset-model](https://pkg.go.dev/github.com/owasp-amass/open-asset-model), there you can find the reference and descriptions to the types.
+
+## Contributing [![Contribute Yes](https://img.shields.io/badge/contribute-yes-brightgreen.svg)](./CONTRIBUTING.md) [![Chat on Discord](https://img.shields.io/discord/433729817918308352.svg?logo=discord)](https://discord.gg/HNePVyX3cp)
```

Comment:
move these up to the top w/ the other GoDoc badge

Reply:
great idea, done 👍